components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts

```diff
@@ -94,6 +94,7 @@ export class CreateUserModalComponent implements OnInit, OnDestroy {
   }
 
   closeCreateModal(): void {
+    this.modifyUsername = false;
```

Comment:
Hi, @vinay033! This is certainly the correct line to add, but all the similar resetting is done in this block:
https://github.com/chef/automate/blob/19339b281fc4400eafb3124c11099b5475606a2e/components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts#L57-L63

I was wondering if there was a reason why you added it in `closeCreateModal` instead? If not, would you mind moving it when you have a chance?

Reply:
I focused on close the modal and then I have added these changes.
but I will create a new one PR to incorporate this.

Reply:
review comment incorporated 
PR: https://github.com/chef/automate/pull/3678